packages/next/src/build/templates/edge-app-route.ts

```diff
@@ -10,6 +10,12 @@ import * as module from 'VAR_USERLAND'
 declare const nextConfig: NextConfigComplete
 // INJECT:nextConfig
 
+const cacheHandlers = {}
+
+if (!(globalThis as any).__nextCacheHandlers) {
+  ;(globalThis as any).__nextCacheHandlers = cacheHandlers
+}
```

Comment:
Is there a reason to put it into a variable or is this sufficient? 

```suggestion
if (!(globalThis as any).__nextCacheHandlers) {
  ;(globalThis as any).__nextCacheHandlers = {}
}
```

Reply:
it's used for replacement in our loadEntrypoints handler